Lesson17/sahra/script.js

```diff
@@ -0,0 +1,107 @@
+// 1. Write a loop that prints out the numbers from 1 to 10.
+
+for (let i = 1; i < 11; i++) {
+  console.log(i);
+}
+
+for (let i = 1; i < 21; i = i + 2) {
+  console.log(i)
+}
+console.log('- - - - - - - - - - - - - - - - - ');
+// 2. (H) Write a loop that prints out the odd numbers from 1 to 20.
+// 1,3,5,7...
+
+for (let i = 0 ; i < 20; i++) {
```

Comment:
This loop prints numbers from 0 to 19, but you need to only print odd numbers from 1 to 20 (1,3,5).